Tasks/google-play-release/task.json

```diff
@@ -104,6 +104,15 @@
             "helpMarkDown": "Checking this box will make the task attempt to update the metadata on your app release"
         },
         {
+            "name": "metadataRootPath",
+            "type": "filePath",
+            "label": "Metadata Root Directory",
+            "defaultValue": "",
```

Comment:
If you don't specify anything, there is no value. If you set no value, the task will fail.

Since you have to check the box to see this field, and the field is marked as required both in the task.json and the usage in the task (fixed now), it's up to the end user to pick the correct directory.

Thus, I am against setting a default value here.